docs/spec/SPEC-SPEC.md

```diff
@@ -0,0 +1,41 @@
+# Specification of Specifications
+
+This file intends to outline the common structure for specifications within
+this directory. 
+
```

Comment:
Can we add a section on tense (see below comment)?

Reply:
Agree on use of present passive tense, adding here